### PR TITLE
Added Gintama openings

### PR DESCRIPTION
**Pull request by vanguard07**
OP 2 to 5

### CODE REVIEW

File: js/data.json [hunk 1]
```diff
@@ -430,6 +430,30 @@ openings = [
...
+	{
+		"link" : "http://67.159.62.2/anime_ost/gintama-op3-single-gin-iro-no-sora/cjbijtbyxz/01%20-%20Gin%20iro%20no%20Sora.mp3",
+		"name" : "Gintama OP3 - Gin iro no Sora",
+		"img"  : "https://myanimelist.cdn-dena.com/s/common/uploaded_files/1449204090-b9875961dca681114b8c47224457c13b.jpeg"
+	},
+
+	{
+		"link" : "http://67.159.62.2/anime_ost/gintama-op4-single-kasanaru-kage/mjydqwsuom/01%20-%20Kasanaru%20Kage.mp3",
+		"name" : "Gintama OP4 - Kasanaru Kage",
+		"img"  : "http://image.tmdb.org/t/p/w780/apO3905dT9eyWnrDtT3BGASziP0.jpg",
```
Comment (aashutoshrathi) [inline js/data.json]: Image have low resolution, may get pixelated.

File: js/data.json [hunk 2]
```diff
@@ -430,6 +430,30 @@ openings = [
...
+	},
+		
+	{
+		"link" : "http://67.159.62.2/anime_ost/gintama-op5-single-donten/ettjaclvre/01%20-%20Donten.mp3",
+		"name" : "Gintama OP5 - Donten",
+		"img"  : "http://i.imgur.com/wnxRZso.png"
```
Comment (aashutoshrathi) [inline js/data.json]: Image have low resolution, may get pixelated.

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]
See Comments

File: js/data.json (hunk 1 above)
Comment (anshumanv) [reply js/data.json]: Yea, try to keep images >720p

**Comment by vanguard07** [comment]
Issues Fixed

**Review by aashutoshrathi — APPROVED** [review]
Quality Seems Great, check for refrences @Anshuman-Verma

**Review by anshumanv — APPROVED** [review]
Looks amazing 👍

**Pull request merged**